### PR TITLE
[bug 1152816] [bug 1152547] Add Firefox for iOS to AAQ

### PR DESCRIPTION
**Pull request by rehandalal**
r?

### CODE REVIEW

File: kitsune/questions/config.py [hunk 1]
```diff
@@ -158,6 +158,45 @@
             }),
         ])
     }),
+    ('ios', {
```
Comment (mythmon) [inline kitsune/questions/config.py]: I'll be so happy when we can ditch all this.

Comment (rehandalal) [inline kitsune/questions/config.py]: +1

**Comment by mythmon** [comment]
This works, but it is hiding a bit of a foot-gun. If we do this and then don't add a `QuestionLocale` object for iOS in English it gets all screwy with infinite redirects. Can we add a migration to add that object if it doesn't exist?

**Comment by rehandalal** [comment]
updated ^

**Comment by mythmon** [comment]
Travis failed, your migration conflicts with the one that landed for the `sr-Cyrl` → `sr` migration. (yay for Travis testing the result of merges!)

Besides that, it looks fine.

**Comment by mythmon** [comment]
Actually, Joni *just* got back to us with a new list of topics to use. Sorry about that. She is adding them to the admin on prod now, so it should work fine.

**Comment by rehandalal** [comment]
fixed ^

**Comment by rehandalal** [comment]
oh, er... still have to add topics.

**Comment by rehandalal** [comment]
updated ^

**Comment by mythmon** [comment]
Travis is super grumpy today.

File: kitsune/questions/migrations/0006_ios_questionlocale.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    Product = apps.get_model('products', 'Product')
+    QuestionLocale = apps.get_model('questions', 'QuestionLocale')
+
+    p = Product.objects.get_or_create(slug='ios', defaults={
+        'title': 'Firefox for iOS',
+        'description': 'Firefox for iPhone, iPad and iPod touch devices',
+        'display_order': 0,
+        'visible': False})
+
+    QuestionLocale.objects.get_or_create(locale='en-US', product=p)
```
Comment (mythmon) [inline kitsune/questions/migrations/0006_ios_questionlocale.py]: Products is a many-to-many field. You'll have to check the return value of get_or_create (which is `obj, was_created`), and add `p` the new object if it was created.

**Comment by rehandalal** [comment]
FIXED FOR REALZ ^

**Comment by mythmon** [comment]
:laughing:

That test assumed we started with zero products, but you added one in the migration.

Man, this patch. It was supposed to be so easy!

**Comment by rehandalal** [comment]
FML! if this doesn't fix it i'm going to lose it.

**Comment by mythmon** [comment]
:tada:. r+, but I think we should hold of merging this until it is closer to Monday, in case we want to land something else between now and then.

**Pull request merged**